guide/interactions/autocomplete.md

```diff
@@ -0,0 +1,110 @@
+# Autocomplete
+
+Slash command options provide the feature of autocomplete. In this section we'll cover how to both add autocomplete support to your command and also how to respond to it.
+
+::: tip
+This page is a follow-up to the [interactions (slash commands) pages](/interactions/registering-slash-commands.md). Please carefully read those first so that you can understand the methods used in this section.
+:::
+
+::: warning
+Make sure that you're on discord.js version 13.3.0 or above before using autocomplete.
+:::
+
+## Preparing commands
+
+To use autocomplete with your command options, you have to tell Discord to enable it.
+How to do this is dependent on how you create your slash commands.
+
+### Slash command builder
+
+The slash command builder does not yet support using autocomplete.
+
+### Custom system
+
+For other custom systems or entering it raw, your resulting code should look something like this. To enable autocomplete, set it to true in the option.
+
+```js {10}
+const commandData = {
+	name: 'autocomplete',
+	description: 'test command to show how autocomplete should be setup',
+	defaultPermission: true,
+	options: [
+		{
+			name: 'name',
+			description: 'Name of something',
+			type: 'STRING',
+			autocomplete: true,
+		},
+	],
+};
+```
+
+## Receiving
+
+To receive a AutocompleteInteraction, you can listen to the `interactionCreate` event and use the `Interaction#isAutocomplete()` method to make sure you only receive autocomplete interactions:
+
+```js {2}
+client.on('interactionCreate', interaction => {
+	if (!interaction.isAutocomplete()) return;
+	console.log(interaction);
+});
+```
+
+## Responding
+
+The AutocompleteInteraction class provides the `AutocompleteInteraction#respond()` method for responding to the interaction.
+
+### Respond with search
+Using `AutocompleteInteraction#respond()` you submit an array of ApplicationCommandOptionChoice objects. Passing an empty array will show "No options match your search" in Discord.
```

Comment:
```suggestion
Using <DocsLink path="class/AutocompleteInteraction?scrollTo=respond" /> you submit an array of <DocsLink path="typedef/ApplicationCommandOptionChoice" /> objects. Passing an empty array will show "No options match your search" for the user.
```

Reply:
Could you add `can` after `<DocsLink path="class/AutocompleteInteraction?scrollTo=respond" /> you` so it ends up as
```md
<DocsLink path="class/AutocompleteInteraction?scrollTo=respond" /> you can submit
```

Reply:
```suggestion
Using <DocsLink path="class/AutocompleteInteraction?scrollTo=respond" /> you can submit an array of <DocsLink path="typedef/ApplicationCommandOptionChoice" /> objects. Passing an empty array will show "No options match your search" for the user.
```